components/org.wso2.carbon.identity.api.user.authorized.apps/org.wso2.carbon.identity.rest.api.user.authorized.apps.v2/src/gen/java/org/wso2/carbon/identity/rest/api/user/authorized/apps/v2/AuthorizedAppsApi.java

```diff
@@ -33,13 +35,13 @@
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.Response;
 
+@Path("/authorized-apps")
+@Api(description = "The authorized-apps API")
 
-@Path("/applications")
-@Api(description = "The applications API")
-public class ApplicationsApi  {
+public class AuthorizedAppsApi  {
 
     @Autowired
-    private ApplicationsApiService delegate;
```

Comment:
How come there's some history of `ApplicationsApiService` in the `AuthorizedAppsApi.java` file?
In your previous implementation you didn't change this file right?